### ISSUE TITLE
Extra reported as being included by default and cannot be removed

### ISSUE THREAD

**Issue by danpascu** [LazyVim/LazyVim#3774]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.11.0-dev-138+g88fe467b1

### Operating system/version

Linux Debian unstable

### Describe the bug

After some recent updates (it appeared since maybe around 2 weeks ago), if I go to LazyExtras I have an entry in the Enabled Plugins list that is marked with an orange dot. I thought maybe some error slipped into the code and it would be removed after a while, but it's still there today so I decided to report it.

This entry is `editor.telescope` and after the dependency list I have a comment following a square dot that reads:
`This extra is included by default`

If I try to press x on it to disable it, I get a notification that says: `Not managed by LazyExtras. Remove from your config to enable/disable here`.

The thing is I cannot find anything in my config related to this. My lazyvim.json looks like this:

```
{
  "extras": [
    "lazyvim.plugins.extras.coding.mini-surround",
    "lazyvim.plugins.extras.coding.yanky",
    "lazyvim.plugins.extras.lang.python",
    "lazyvim.plugins.extras.lang.toml",
    "lazyvim.plugins.extras.util.project"
  ],
  "news": {
    "NEWS.md": "6077"
  },
  "version": 6
}
```

and if I search for editor or telescope in ~/.config/nvim I only get this:

```
$ rg editor ~/.config/nvim
~/.config/nvim/LICENSE
42:      editorial revisions, annotations, elaborations, or other modifications

~/.config/nvim/lua/plugins/ui.lua
28:        -- * a percentage of the width / height of the editor when <= 1

$ rg telescope ~/.config/nvim
~/.config/nvim/lazy-lock.json
40:  "telescope-fzf-native.nvim": { "branch": "main", "commit": "9ef21b2e6bb6ebeaf349a0781745549bbb870d27" },
41:  "telescope.nvim": { "branch": "master", "commit": "f2bfde705ac752c52544d5cfa8b0aee0a766c1ed" },

```

### Steps To Reproduce

1. start nvim and go to extras

### Expected Behavior

I expect the orange dot not to be there, or at least find something in my config related to it and be able to remove it from there.

**Comment by danpascu** [comment]
I just tried this with a fresh install of LazyVim/starter:

```
# ~/.config/nvim.test doesn't exist
git clone https://github.com/LazyVim/starter ~/.config/nvim.test
NVIM_APPNAME=nvim.test nvim
```

and after it finishes installing and configuring all of its dependencies, when I go to LazyExtras there is 1 entry in the `Enabled Plugins` section that reads `editor.telescope`, is marked with the orange dot and cannot be disabled.

**Comment by dpetka2001** [comment]
Yes Telescope is included by default. Otherwise there would not be any fuzzy finder. You can enable `editor.fzf` if you want and that will disable Telescope. This is not a bug. It's just the way Telescope was moved to the Extras by the maintainer.

**Comment by danpascu** [comment]
You completely misunderstood what I said.

My complaint wasn't about what fuzzy finder is the default, nor about what is included in the core and what in extra. I also do NOT want to get rid of telescope or replace it with fzf.

My problem is that a core plugin is somehow also listed in extras with an error message attached to it and I can't get rid of this superfluous entry from extras. See the image below. So if it's included by default, why is it listed in extras with an error message saying that it should't be there?

![lazyvim-error](https://github.com/LazyVim/LazyVim/assets/4569572/4b869d69-0eb1-433d-b98b-cb9e59c49393)

**Comment by dpetka2001** [comment]
Because Telescope was indeed moved to the LazyExtras instead of being a core plugin and is enabled by default in the Extras as the default fuzzy finder. That's why you see this. Either way nothing to worry about. It's normal.

**Comment by danpascu** [comment]
In this case, I have to say, the way it is presented is very confusing. When I first interacted with this, the message I got from it was that it was some error and the problem needed to be fixed somewhere in my configuration.

I know understand from your comments that this is an extra, but is always enabled.

I this case may I suggest maybe a change in the way is presented. I would suggest the message

`This extra is included by default` to be replaced by `This extra is enabled by default`

The first message suggested to me that this module was no longer an extra, but now was included by default as a core plugin, but somehow the entry was still left in the extra list. Also the orange bullet suggested that there was a problem with it, like it was listed there by error.

The second suggestion would be to update the notification message when one tries to press `x` to disable the plugin from:

`Not managed by LazyExtras. Remove from your config to enable/disable here.`

to something like:

`This plugin cannot be disabled. You may enable a different alternative like editor.fzf to replace it`


The fact that the notification said `Not managed by LazyExtras. ...` directly suggested to me that the plugin was not part of extras and it was listed there by error. The combination of messages and notifications sent a completely different messages that what is intended.

**Comment by dpetka2001** [comment]
The message `Not managed by LazyExtras` means that it's not enabled by pressing `x` in the LazyExtras UI. It's enabled by using a `import = lazyvim.plugins.extras.editor.telescope`, that's why you're seeing that message. If you had other Extras in the past using the `import` feature, you would see the same message. Only the Extras that are enabled by pressing `x` in the LazyExtras are managed by LazyExtras, not the ones that are enabled by using `import`. That's just the way the implementation is.

`This extra is included by default` vs `This extra is enabled by default` has exactly the same meaning with just different wording. Anyway, not worth debating over this any longer.

**Comment by danpascu** [comment]
There is a difference between included and enabled. Enabled is clear, it's just an yes/no flag. Included is unclear if it means included in the list of extras, or included in LazyVim as a whole.

We can argue semantics all day long, but in the end the point is that I had a bad experience interacting with this, because it did not convey its intent well (IMO). I'm sure I'm not the only one that would interpret it this way:

1. I see an orange dot that seems to suggest something is not right.
2. The message next to it says `This extra is included by default` which I take to mean that it's no longer an extra but it was moved to be part of the core set of plugins.
3. I'm not sure why is it still listed in extras, but I conclude that since it's now included by default in the core, I might as well remove the superfluous entry from extras. The orange dot seems to suggest it shouldn't be there.
4. I press x to be greeted with `Not managed by LazyExtras. Remove from your configs to enable/disable here.` which further reinforces the idea that this is not an extra anymore, but a core plugin now and maybe something in my configuration makes it still stick around in the extras list.
5. I check my configuration to no avail as there is no reference to it in my config directory and I conclude that it's a bug.

I don't find my line of thinking unreasonable and I expect other people may interpret things the same way, hence I suggested some changed to how it's presented.

If you think that the UI expresses things just fine and I'm the odd one out that couldn't understand the clear meaning, then feel free to ignore the suggestions.

**Comment by windowsrefund** [comment]
100% agree with OP here as I just hit this same point of confusion. All I'm trying to accomplish is to manage my own fzf-lua config in order to avoid inheriting the one that is baked into the distro. Granted, that may be easier than I realize but the documentation and confusing messages related to `vim.g.lazyvim_picker` seem a bit too much?

**Comment by vlaw** [comment]
After seeing the message 'This extra is included by default', I pressed `x` too, to try removing the extra configuration. 
I thought the message meant I had already enabled this extra and LazyVim was trying to enable it twice. 
Since the prompt indicated there was a duplicated configured extra, it would be better to remove it from my own config file. Additionally, I even opened `lazyvim.json` trying to locate that line and delete it manually. 

In the end, it's really confusing.
